opendbc/safety/tests/test_hyundai_canfd.py

```diff
@@ -161,6 +165,14 @@ def test_button_sends(self):
         self.safety.set_controls_allowed(enabled)
         self.assertFalse(self._tx(self._button_msg(btn)))
 
+  def test_acc_cancel(self):
+    # FIXME: the CANFD_ALT_BUTTONS cars are the only ones that use SCC_CONTROL to cancel, why can't we use buttons?
```

Comment:
@sunnyhaibin know anything about this?

Reply:
@sunnyhaibin can you test sending the buttons to simplify this?

Reply:
@sshane For cars with only `CRUISE_BUTTONS_ALT`/`0x1AA`, I have tried sending the `PAUSE/RESUME` button via `CRUISE_BUTTONS_ALT`/`0x1AA` (bus 0, 1, 2, respectively), and the car did not respond at all to our cancel command.